pkg/registries/adapters/local_openshift_adapter.go

```diff
@@ -0,0 +1,137 @@
+//
+// Copyright (c) 2017 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Red Hat trademarks are not licensed under Apache License, Version 2.
+// No permission is granted to use or replicate Red Hat trademarks that
+// are incorporated in this software or its documentation.
+//
+
+package adapters
+
+import (
+	logging "github.com/op/go-logging"
+	"github.com/openshift/ansible-service-broker/pkg/apb"
+	"github.com/openshift/ansible-service-broker/pkg/clients"
+	yaml "gopkg.in/yaml.v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+)
+
+const localOpenShiftName = "openshift-registry"
+
+// LocalOpenShiftAdapter - Docker Hub Adapter
+type LocalOpenShiftAdapter struct {
+	Config Configuration
+	Log    *logging.Logger
+}
+
+// RegistryName - Retrieve the registry name
+func (r LocalOpenShiftAdapter) RegistryName() string {
+	return localOpenShiftName
```

Comment:
I think the name can be changed of the registry. Think this should be like rhcc: ```return r.Config.URL.String()```

Reply:
The internal registry uses the api. Name will always be the same.